src/definitions/elements/label.less

```diff
@@ -160,6 +160,7 @@ a.ui.label {
 
 /* Padding on next content after a label */
 .ui.top.attached.label:first-child + :not(.attached),
+.ui.top.attached.label:nth-of-type(2) + :not(.attached),
```

Comment:
I suggest to change the whole to

```css
  .ui.top.attached.label ~ :not(.attached) {
    margin-top: 2rem !important;
  }
  .ui.bottom.attached.label ~ :not(.attached) {
    margin-top:0;
    margin-bottom: 2rem !important;
  } 
```

That would also cover `.ui-bottom.attached.label` left and right then. 😉 

See your adjusted jsfiddle here https://jsfiddle.net/pqny178m/
 
![image](https://user-images.githubusercontent.com/18379884/59144539-9f000d00-89d9-11e9-8bae-593306b28c11.png)

Reply:
The following is even better, otherwise any element will get the margin:

```css
  .ui.top.attached.label ~ :first-of-type:not(.attached) {
    margin-top: 2rem !important;
  }
  .ui.bottom.attached.label ~ :last-child:not(.attached) {
    margin-top:0;
    margin-bottom: 2rem !important;
  }
```
See https://jsfiddle.net/pqny178m/1/

Reply:
WFM.

Reply:
Unfortunately the provided (for you working) change is not working anymore when an additional `bottom attached` label is added.
See https://jsfiddle.net/v9dgfn5y/

![image](https://user-images.githubusercontent.com/18379884/59147950-4db84380-8a02-11e9-954f-0543313d7f89.png)
My suggestion instead works for this case also https://jsfiddle.net/v9dgfn5y/1/
![image](https://user-images.githubusercontent.com/18379884/59148010-ef3f9500-8a02-11e9-8837-24b12c4981ea.png)

I appreciate the time you spent improving FUI. However, if you want your PR to get merged, you should consider a consistent solution which should work for everybody and not only for yourself  :wink:

Reply:
Hey - sorry, that's not what I meant - I was under the impression that you could apply your change to my PR (as a maintainer) - and that's fine with me. If you'd prefer me to update the PR, I'm happy to do that too.

Reply:
😄 Alright, then sorry for the misunderstanding. Yes, we usually prefer the authors to update their PRs on their own. 🙂 They of course should always agree to and/or discuss reviewers suggestions, because our suggestions might be wrong, so i advise to test reviewers suggestions also critically before blindly apply the changes. There might be situations i also have overseen